src/components/CV_hierarchy/hierarchy.js

```diff
@@ -58,10 +58,10 @@ const CreateTree = ({ data, onClick, countMap, renderZeroCount }) => {
       }
       else {
         if (val["children"].length === 0) {
-          return <StyledTreeItem key={label} nodeId={label} label={labelContent(label, labelCnt)} onClick={onClick} />;
+          return <StyledTreeItem key={label} itemId={key} label={labelContent(label, labelCnt)} onClick={onClick} />;
         } else {
           return (
-            <StyledTreeItem key={label} nodeId={label} label={labelContent(label, labelCnt)} onClick={onClick}>
+            <StyledTreeItem key={label} itemId={key} label={labelContent(label, labelCnt)} onClick={onClick}>
```

Comment:
Is the switch from `nodeId` to `itemId` because of the change in the component version?

Reply:
Yes it is due to the changes in the `TreeView` package, now `itemId` is used instead of `nodeId` and it should be a unique value. The [migration documentation](https://mui.com/x/migration/migration-tree-view-v6/) includes this change.